examples/inference/serving/torch_serve/Colossal_Inference_Handler.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import os
+import random
+import socket
+import zipfile
+from abc import ABC
+
+import torch
+import transformers
+from transformers import AutoTokenizer, BloomForCausalLM, BloomTokenizerFast, LlamaForCausalLM
+from ts.torch_handler.base_handler import BaseHandler
+
+import colossalai
+from colossalai.inference.tensor_parallel.engine import TPInferEngine
+from colossalai.shardformer import ShardConfig
+
+logger = logging.getLogger(__name__)
+logger.info("Transformers version %s", transformers.__version__)
+logger.info("ColossalAI version %s", colossalai.__version__)
+
+
+# from colossalai.testing
+# assins a random port, for demo use only
+def free_port() -> int:
```

Comment:
why not import from colossalai directly?

Reply:
Revised. Now imports from colossalai.testing